bigtable/admin_test.go

```diff
@@ -56,15 +66,34 @@ func setupTableClient(t *testing.T, ac btapb.BigtableTableAdminClient) *AdminCli
 	return c
 }
 
-func TestTableAdmin_UpdateTable(t *testing.T) {
+func TestTableAdmin_CreateTableFromConf(t *testing.T) {
```

Comment:
I usually use a parameterized test in C++ to test both Protected and Unprotected cases. I am not sure how this can be done in go.